### ISSUE TITLE
accuracy about classification

### ISSUE THREAD

**Issue by grxiao** [WangYueFt/dgcnn#35]
Hi,when I run the tensorflow code.I just got the accuracy of 91.2% .I read the paper published in 2018,the result is as sama sa the baseline .I want to the resaon.thanks!

**Comment by grxiao** [comment]
@WangYueFt hi！Could you provide the improved version code under tensorflow which get the result of 92.2%?

**Comment by cookie54sun** [comment]
I also got classification acc 91.2% with tensorflow code.  GPU is GTX1080.

**Comment by cookie54sun** [comment]
I used the model  pretrained by pytorch and got 92.9 acc on ModelNet40. I think the reason is that author replaced relu to leakyrelu in pytorch code and so on, like what the author said in [the earlier issues.](https://github.com/WangYueFt/dgcnn/issues/22) Maybe there's no change in the tensorflow code.

**Comment by grxiao** [comment]
yes,I think the tensorflow code is mainly for the segmentation task.



---Original---

**Comment by WangYueFt** [comment]
Yeah. We use the updated version (pytorch) for classification in our new draft.
